src/test/java/duke/ParserTest.java

```diff
@@ -0,0 +1,124 @@
+package duke;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class ParserTest {
+
+    @Test
+    public void testIsCreateTaskCommandTodo() {
```

Comment:
Perhaps the test method name should follow the format: "...Test", where "..." represent the original method name.